clients/src/main/java/org/apache/kafka/common/Uuid.java

```diff
@@ -27,6 +27,10 @@
  */
 public class Uuid implements Comparable<Uuid> {
 
+    /**
+     * A UUID for the metadata topic in KRaft mode. Will never be returned by the randomUuid method.
+     */
+    public static final Uuid SENTINEL_ID = new Uuid(0L, 1L);
```

Comment:
Could we be more explicit and call this `METADATA_TOPIC_ID` or something like that? Do we have other contexts where we use it?

Reply:
As of now, no. I'm good with changing it.